src/components/Header.tsx

```diff
@@ -1,31 +1,152 @@
+import { HiBell, HiMenu, HiX } from 'react-icons/hi';
 import Link from 'next/link';
-import { FaDiscord, FaGithub, FaTwitter } from 'react-icons/fa';
+
+function classNames(...classes: string[]) {
+	return classes.filter(Boolean).join(' ');
+}
 
 export function Header() {
 	return (
-		<nav className='flex items-center justify-between py-4 shadow-[rgba(17,17,26,0.1)_0px_1px_0px] lg:pl-[20%] lg:pr-[20%]'>
-			<Link href='/'>
-				<a>
-					<span className='cursor-pointer px-5 text-3xl font-bold text-gray-700'>defaang</span>
-				</a>
-			</Link>
-			<div className='mx-5 flex w-1/3 items-center justify-between lg:w-1/6'>
-				<Link href='https://github.com/ykdojo/defaang'>
-					<a target='_blank' rel='noopener noreferrer'>
-						<FaGithub className='h-8 w-8' title='GitHub' />
-					</a>
-				</Link>
-				<Link href='https://discord.gg/aJp6Fypb'>
-					<a target='_blank' rel='noopener noreferrer'>
-						<FaDiscord className='h-8 w-8' title='Discord' />
-					</a>
-				</Link>
-				<Link href='https://twitter.com/ykdojo'>
-					<a target='_blank' rel='noopener noreferrer'>
-						<FaTwitter className='h-8 w-8' title='Twitter' />
-					</a>
-				</Link>
-			</div>
-		</nav>
+		<Disclosure as='nav' className='bg-gray-800'>
+			{({ open }) => (
+				<>
+					<div className='mx-auto max-w-7xl px-4 sm:px-6 lg:px-8'>
+						<div className='flex h-16 items-center justify-between'>
+							<div className='flex items-center'>
+								<div className='flex-shrink-0'>
+									<img
+										className='block h-8 w-auto lg:hidden'
+										src='https://tailwindui.com/img/logos/workflow-mark.svg?color=indigo&shade=500'
+										alt='Workflow'
+									/>
+									<img
+										className='hidden h-8 w-auto lg:block'
+										src='https://tailwindui.com/img/logos/workflow-mark.svg?color=indigo&shade=500'
+										alt='Workflow'
+									/>
+								</div>
+								<div className='hidden sm:ml-6 sm:block'>
+									<div className='flex space-x-4'>
+										<a href='#' className='rounded-md bg-gray-900 px-3 py-2 text-sm font-medium text-white'>
+											Dashboard
+										</a>
+										<a
+											href='#'
+											className='rounded-md px-3 py-2 text-sm font-medium text-gray-300 hover:bg-gray-700 hover:text-white'
+										>
+											Team
+										</a>
+										<a
+											href='#'
+											className='rounded-md px-3 py-2 text-sm font-medium text-gray-300 hover:bg-gray-700 hover:text-white'
+										>
+											Projects
+										</a>
+										<a
+											href='#'
+											className='rounded-md px-3 py-2 text-sm font-medium text-gray-300 hover:bg-gray-700 hover:text-white'
+										>
+											Calendar
+										</a>
+									</div>
+								</div>
+							</div>
+							<div className='hidden sm:ml-6 sm:block'>
+								<div className='flex items-center'>
+									<Link href='/signin'>
+										<a className='text-white' target='_blank' rel='noopener noreferrer'>
```

Comment:
This shouldn't open in a new tab. Also, there is no need for a `rel` attribute here as it's an internal link.
```suggestion
										<a className='text-white'>
```